README.md

```diff
@@ -170,41 +111,98 @@ We also provide network-specific example env configuration files in `.env.exampl
+
+### Retrieving Blobs
 
-The `raw commitment` is an RLP-encoded [EigenDA certificate](https://github.com/Layr-Labs/eigenda/blob/eb422ff58ac6dcd4e7b30373033507414d33dba1/api/proto/disperser/disperser.proto#L168).
+Validator nodes proceed with the exact reverse process as that used by the sequencer in the [posting blobs](#posting-blobs) section. The rollup validator submits a GET request to the proxy with the DACert in the body. The proxy validates the cert, fetches the corresponding blob from EigenDA, validates it, decodes it back into the rollup payload, and returns it the rollup node.
 
-**NOTE:** Commitments are cryptographically verified against the data fetched from EigenDA for all `/get` calls. The server will respond with status `500` in the event where EigenDA were to lie and provide falsified data thats irrespective of the client provided commitment. This feature cannot be disabled and is part of standard operation.
+### Rollup Commitment Schemas
+
+> Warning: the name `commitment` here refers to the piece of data sent to the rollup's batcher inbox (see op spec's [description](https://specs.optimism.io/experimental/alt-da.html#input-commitment-submission)), not to blobs' KZG commitment. The Rollup commitment consists of a few-byte header (described below) followed by a `DACertificate`, which contains all the information necessary to retrieve and validate an EigenDA blob. The `DACertificate` itself contains the KZG commitment to the blob.
+
+Currently, there are two commitment modes supported with unique encoding schemas for each. The `version byte` is shared for all modes and denotes which version of the EigenDA `DACertificate` is being used/requested. The following versions are currently supported:
+* `0x00`: EigenDA V1 certificate type (i.e, dispersal blob info struct with verification against service manager)
+* `0x01`: EigenDA V2 certificate type
+
+#### Optimism Commitment Mode
+For `alt-da` Optimism rollups using EigenDA, the following [commitment schemas](https://specs.optimism.io/experimental/alt-da.html#example-commitments) are supported by our proxy:
+
+| commitment_type (byte) | da_layer_byte | version_byte | payload           |
+| ---------------------- | ------------- | ------------ | ----------------- |
+| 0x00                   |               |              | keccak_commitment |
+| 0x01                   | 0x00          | 0x00         | eigenda_cert_v1   |
+| 0x01                   | 0x00          | 0x01         | eigenda_cert_v2   |
+
+`keccak256` (commitment_type 0x00) uses an S3 storage backend with where a simple keccak commitment of the `blob` is used as the key. For `generic` commitments, we (obviously) only support da_layer_byte 0x00 which represents EigenDA.
```

Comment:
why "(obviously)"?

Reply:
da_layer_byte is used in op-stack to refer to which da-layer to retrieve the blob from. Given that this repo is called EIGENDA proxy, we "obviously" only support this network.

But perhaps there's a clearer way to rephrase this? Or shall I just remove completely?

Reply:
I think I'd *either* remove the whole sentence, or remove the word "obviously". If it must be said, then it must not be perfectly obvious (knowing nothing, it wasn't obvious to me). But if it _is_ perfectly obvious, then no need to say it.

This is really nitpicky comment, though, you can feel free to just ignore and resolve if you prefer

Reply:
No I think you're right. Removed in [7b8d9d6](https://github.com/Layr-Labs/eigenda-proxy/pull/233/commits/7b8d9d62a6e4ed46947de73df0a966a99ce513b9)